dev.py

```diff
@@ -426,17 +427,63 @@ def install_project(show_build_log):
             raise
     elapsed = datetime.datetime.now() - start_time
 
-    if ret == 0:
-        print("Installation OK")
-    else:
+    if ret != 0:
         if not show_build_log:
             with open(log_filename, 'r') as f:
                 print(f.read())
         print("Installation failed! ({0} elapsed)".format(elapsed))
         sys.exit(1)
+
+    # On Windows we also need openblas lib installed
+    if platform.system() == 'Windows':
+        ret = copy_openblas()
+    if ret != 0:
+        print("OpenBLAS copy failed!")
+        sys.exit(1)
+    print("Installation OK")
     return
 
 
+def import_module(mod_name, mod_path):
+    # https://bugs.python.org/msg255901
+    import importlib.util as impu
+    spec = impu.spec_from_file_location(mod_name, mod_path)
+    mod = impu.module_from_spec(spec)
+    spec.loader.exec_module(mod)
+    return mod
```

Comment:
This is useful to put in a function, because there is a second usage already in this file. Can you refactor that and remove the duplicate import?